meetings/2022-11-30.md

```diff
@@ -40,19 +57,100 @@
+* SC: The idea is that if a server offers multiple storages, should the owners of each storage have trust between them? There were no objections, one approval, but there were comments which requested clarity on the intention. I think we need better justification/language to clarify those concerns, or we don't include in next release. 
+* SC: This was worked on with TBL, KK, JB, and myself. Maybe others can offer more insight. 
+* TBL: Very obscure point, important but "trust between..." is ambiguous. Does it mean you should not build systems where there is no trust? Does it mean that there will be trust between ??? necessarily? The way it is stated is very confusing, not helpful.
+* SC: Good self-review because we drafted together :) 
+* TBL: Yes, and I thought that when we did.
+* SC: First part of this is a note. Second part is ??? That is the one where commenter felt it was unclear. If we focus on that we can move this forward. Line 904.
+* TBL: ??? about web app security? If people have have webpages with JS on their pods, the issue ??? Solid doesn't use web origins, but when people use solid pods are ordinary blogs, for example.. 
+* SC: Pre-proposal: would you (TBL) or anyone object to not including that statement at all?
+* TBL: Happy to leave the whole PR out of next release.
+* SC: No objections to the note and people find it useful. It is the second part. I can update PR to remove that. Would you approve?
+* TBL: Yes. 
+
+ACTION: SC to update PR to reflect the discussion (remove second part, leave note).
+
+
+### Agent-specific Discovery
+URL: https://woutermont.github.io/solid/agent-specific-discovery.html
+
+* WT: Proposal based on stuff already in SAI, but tries to take out a specific part that doesn't belong with the rest, and make it a separate specification. It is concerned about agents/client who want to discover info about some entity, can find not only public info available in identity document, but they can also find an easy way to know where the info specific to that client lives. 
+* WT: One approach would be with a lot of seeAlsos in profile document and have client check all of them. ??? proposes to have a microservice/hub to which an entity points to within their profile doc, and client can ask the service for a pointer to the place where their specific data can be found. 
+* SC: How would you like to continue with this proposal?
+* WT: I want to bring awareness. I'd like people to comment on it to see if it's valuable as a separate specification. Would like to invite SAI and WebID Profile editors to react as how they see it fits next to or in line with their proposals. 
+* VB: We can discuss next WebID-profile meeting. Will make sure to read thought and prepare questions/comments, so that it's productive.
```

Comment:
```suggestion
* VB: We can discuss this at the next WebID-profile meeting. Will make sure to read through and prepare questions/comments, so that it's productive.
```